src/main/java/ui/MenuControl.java

```diff
@@ -336,9 +334,11 @@ private MenuItem createRefreshMenuItem() {
     private void updateRepoRemoveList(Menu remove) {
         remove.getItems().clear();
 
-        Set<String> currentlyUsedRepos = ui.getCurrentlyUsedRepos();
-        Set<String> removableRepos = ui.logic.getStoredRepos();
-        removableRepos.removeAll(currentlyUsedRepos);
+        Set<String> currentlyUsedRepos = ui.getCurrentlyUsedRepos()
+                .stream().map(String::toLowerCase).collect(Collectors.toSet());
+        Set<String> removableRepos = ui.logic.getStoredRepos()
+                .stream().filter(repoId -> !currentlyUsedRepos.contains(repoId.toLowerCase()))
+                .collect(Collectors.toSet());
```

Comment:
If `getCurrentlyUsedRepos` already provides some guarantees, are all the extra `toLowerCase` necessary?